v2/googlecloud-to-neo4j/src/test/java/com/google/cloud/teleport/v2/neo4j/model/connection/ConnectionParamsTest.java

```diff
@@ -154,14 +168,15 @@ public void parsesFullCustomInformation() throws Exception {
     ConnectionParams connectionParams =
         Json.map(
             json(
-                "{\"auth_type\": \"custom\", \"server_url\": \"bolt://example.com\", \"database\": \"db\", \"principal\": \"a-principal\", \"credentials\": \"some-credentials\", \"realm\": \"a-realm\", \"scheme\": \"a-scheme\", \"parameters\": {\"foo\": \"bar\", \"baz\": true}}"),
+                "{\"auth_type\": \"custom\", \"server_url\": \"bolt://example.com\", \"database\": \"db\", \"custom_certificate_path\": \"gs://example.com/my/cert\", \"principal\": \"a-principal\", \"credentials\": \"some-credentials\", \"realm\": \"a-realm\", \"scheme\": \"a-scheme\", \"parameters\": {\"foo\": \"bar\", \"baz\": true}}"),
             ConnectionParams.class);
 
     assertThat(connectionParams)
         .isEqualTo(
             new CustomConnectionParams(
                 "bolt://example.com",
                 "db",
+                "gs://example.com/my/cert",
```

Comment:
nit: could we also make it clear that this is not `my` but `ca` certificate?

```suggestion
                "gs://example.com/ca/cert",
```

Reply:
fixed in 1242a4af7c76efc8152e02a52ef3250881652d80